src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -43,6 +43,11 @@ def get(self, request):
         if not next_url:
             raise DisallowedRedirect
 
+        # This is needed for the next steps in order to redirect the user to the right place again.
+        # When a user is authenticated and cancels the login flow, the next url (oidc_login_next)
+        # has been removed from the session (the user is logged out-session flushed).
+        self.request.session["of_redirect_next"] = next_url
```

Comment:
I'm not a 100% certain, but: shouldn't session be saved?

Reply:
Do you mean generally?If so, django does this when the session is modified (afaik). Let me know if you mean something like that [https://docs.djangoproject.com/en/5.0/topics/http/sessions/#when-sessions-are-saved](url) or if you mean something else.

Reply:
Yes indeed. In tests I have to save it as you mention. Maybe @sergei-maertens  can provide us with a more clear answer.

Reply:
the remarks above are correct :)